operator/validator/controller.go

```diff
@@ -1242,38 +1256,31 @@ func (c *controller) startCommittee(vc *validator.Committee) (bool, error) {
 	return true, nil
 }
 
-func (c *controller) ForkMonitor(logger *zap.Logger) {
+func (c *controller) ForkListener(logger *zap.Logger) {
 	if c.networkConfig.PastAlanFork() {
 		return
 	}
 
-	bn := c.beacon.GetBeaconNetwork()
-
-	forkTime := time.Unix(bn.EstimatedTimeAtSlot(bn.FirstSlotAtEpoch(c.networkConfig.AlanForkEpoch)), 0)
-	timeUntilFork := time.Until(forkTime)
+	slotTicker := slotticker.New(c.logger, slotticker.Config{
+		SlotDuration: c.networkConfig.SlotDurationSec(),
+		GenesisTime:  c.networkConfig.GetGenesisTime(),
+	})
 
+	next := slotTicker.Next()
 	for {
 		select {
-		case <-c.context.Done():
-			return
-		case <-time.After(timeUntilFork):
-			logger.Info("Stopping genesis validators after alan fork")
-			c.CleanGenesisValidators()
-			logger.Info("✅Successfully stopped genesis validators")
```

Comment:
@moshe-blox, why did you remove the logs here? It happens once and bring some confidence stopping did happen

Reply:
agree, i returned the log